src/components/templates/UiQuestion/UiQuestion.vue

```diff
@@ -200,7 +205,11 @@ const props = defineProps({
     default: () => ({
       info: false,
       why: false,
-      issue: { feedback: false },
+      issue: {
+        action: false,
+        feedback: false,
+        skip: true,
```

Comment:
Why is this one `true` by default?

Reply:
'skip' is visible on designs and by default is enabled

Reply:
But both `feedback && action` are `false`, so `skip` won't be available anyway. 🤔

Reply:
Yes, because it does not depend on `feedback` and `action`. 
You will use `action` to display button. 
When you change `feedbeck` you will see `Notification`. By default, this notification will display the Skip button (only when you provide onClick for it). But. When you set skip to false and provide onClick this button will be hidden.

Reply:
If you have `feedback` and `action` off, then `skip` doesn't do anything, regardless if it's on or off. 🤔